packaging/build/libyaml.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eux
+
+. ./LIBYAML_VERSION
+TD="$(mktemp -d)"
+pushd "$TD" || exit 1
+git clone https://github.com/yaml/libyaml.git
+pushd libyaml
+git config --global advice.detachedHead false
+git reset --hard "$LIBYAML_VERSION"
+./bootstrap
+./configure --disable-dependency-tracking
```

Comment:
This should force the libyaml build to only generate a static lib, which is absolutely required for useful wheels:
```suggestion
./configure --disable-dependency-tracking --with-pic --enable-shared=no
```